src/store/eventing/staged.ts

```diff
@@ -39,12 +35,11 @@ export const handleSetStagedAction = (prevState: AppState, nextState: AppState):
     const prevStaged = getCreatorStaged(prevState);
     const nextStaged = getCreatorStaged(nextState);
     const status = getStatus(prevStaged, nextStaged);
-    const type = getType(prevStaged) || getType(nextStaged);
-
-    if (!status || !type) {
+    const mode = getAnnotationMode(nextState);
+    const type = getType(prevStaged) || getType(nextStaged) || (mode === Mode.NONE ? null : mode);
```

Comment:
Why is `Mode` involved here? Should the consumer (Preview SDK) track the mode and choose whether to respond to the event or not, instead?

Reply:
`Mode` is involved here because in the current implementation, `getType(prevStaged)` and `getType(nextStaged)` both return null, which results in `type` being set to be `null`. Since `type` is `null`, in the following if check, we end the code execution rather than emitting an event.

Since we don't emit an event, then we aren't able to set the toggle state of the region button back to be NONE state.

Reply:
Also, `Mode` in this case would be `region`, so we would end up emitting the event.